pkg/roachprod/vm/gce/dns.go

```diff
@@ -29,7 +29,7 @@ import (
 const (
 	dnsManagedZone           = "roachprod-managed"
 	dnsDomain                = "roachprod-managed.crdb.io"
-	dnsMaxResults            = 1000
+	dnsMaxResults            = 10000
```

Comment:
How does this correspond to the API limit? For [list](https://cloud.google.com/sdk/gcloud/reference/dns/record-sets/list), it seems to default to `unlimited`,

> --limit=LIMIT
Maximum number of resources to list. The default is unlimited. This flag interacts with other flags that are applied in this order: --flatten, --sort-by, --filter, --limit.

Reply:
Interesting that `--limit` being applied _after_ `--filter` is documented behaviour. I was talking to @herkolategan about this earlier. Easy to overlook this (i.e., this is not a `SELECT ... LIMIT` 🙂)

Reply:
> How does this correspond to the API limit

That `--limit=LIMIT` gets set to `dnsMaxResults`
I should have been clearer, that the API call in my commit msg refers to the HTTP API call, not the CLI. I'll update my commit msg.

> it seems the API does indeed filter before applying the limit? In that sense, do we need to bump the limit?

That's what I pasted results for in the chat, it seems to be the other way around, maybe a bug in the CLI tool then. I'll double check.